lib/simple_assert.ex

```diff
@@ -24,4 +24,29 @@ defmodule SimpleAssert do
       raise ArgumentError, "assertion failed!"
     end
   end
+
+  @doc """
+  Asserts the `e` is `false` or `nil`.
+
+  If it succeeds, it returns `:ok`. Raises an error otherwise.
+
+  ## Examples
+
+      iex> SimpleAssert.exists_false(false)
+      :ok
+
+      iex> SimpleAssert.exists_false(nil)
+      :ok
+
+      iex> SimpleAssert.exists_false(:unexpected_result)
+      ** (ArgumentError) assertion failed!
+  """
+  @spec exists_false(any()) :: :ok | none()
+  def exists_false(e) do
```

Comment:
`assert_false` is better than `exists_false`.

See here: https://blog.emattsan.org/entry/20091004/1254638022

Reply:
Thanks for your comment! I renamed it! https://github.com/miolab/simple_assert/pull/3/commits/743371fdf76b32988d2ed1e7d1f6fa4e63663aee